lib/tapioca/gem/pipeline.rb

```diff
@@ -355,7 +355,7 @@ def defined_in_gem?(constant, strict: true)
       def get_file_candidates(constant)
         wrapped_module = Pry::WrappedModule.new(constant)
 
-        wrapped_module.candidates.map(&:file).to_a.compact
+        wrapped_module.send(:method_candidates).flatten.map(&:source_file).uniq.compact
```

Comment:
Tiny, probably out of scope suggestion, but you should be able to remove the `compact` pass with: 
```suggestion
        wrapped_module.send(:method_candidates).flatten.filter_map(&:source_file).uniq
```
https://blog.saeloun.com/2019/05/25/ruby-2-7-enumerable-filter-map.html

Reply:
Oh yeah, I keep forgetting that that method exists now. Thanks for the suggestion.